File: Corte1/C++/Parcial/Calculadora.cpp
#ifndef CALCULADORA_CPP
#define CALCULADORA_CPP

class Calculadora {
public:
    // Método que procesa un arreglo de notas para devolver el promedio
    float obtenerPromedio(float notas[], int cantidad) {
        float suma = 0;
        // REQUERIMIENTO: Uso de ciclos para recorrer estructuras
        for (int i = 0; i < cantidad; i++) {
            suma += notas[i];
        }
        return (cantidad > 0) ? (suma / cantidad) : 0.0f;
    }
};
#endif

File: Corte2/Python/Evaluación/Calculadora.py
class Calculadora:
    # Método que procesa un arreglo de notas para devolver el promedio
    def obtener_promedio(self, notas, cantidad):
        suma = 0.0
        # REQUERIMIENTO: Uso de ciclos para recorrer estructuras
        for i in range(cantidad):
            suma += notas[i]
        
        return (suma / cantidad) if cantidad > 0 else 0.0